integration-tests/run_tests

```diff
@@ -25,8 +25,9 @@ fi
 
 cleanup=0
 
-if [[ ! -v TEMP_RESOURCES_DIR ]]; then
-	TEMP_RESOURCES_DIR=$(mktemp --tmpdir -d integration-tests.XXXXXXXXXX)
+TMPDIR="/tmp"
+if [[ -z ${TEMP_RESOURCES_DIR+x} ]]; then
+	TEMP_RESOURCES_DIR=$(mktemp -d "${TMPDIR:-/tmp}/integratopn-tests.XXXXXXXXX")
```

Comment:
TMPDIR env var is required on OS X because default folder `mktemp` cannot be used by docker by default.
`+x` makes sure condition fails if `TEMP_RESOURCES_DIR ` is set but null. I've seen it somewhere as some more reliable way to detect if var is unset.

Reply:
Can we at least use relative folder inside project? This is very invasive to create new folder on the root. It's not what would you expect when running a test.

Reply:
I'm not sure I follow - `mktemp` by default creates in `/tmp`, so nothing changes for linux, its only makes it work for OS X. `TEMP_RESOURCES_DIR ` some random folder in /tmp/integration-tests. Actually I have a typo here

Reply:
Location of the temp folder is user configuration and can be any folder not only `/tmp`. So IMO, creating `/tmp` folder in the root of the drive may not be what user expects.
I'll agree that this is not so important so I'll approve this PR.

Reply:
But `/tmp` is expected to exist in any POSIX OS and server as temp data, do I miss smth?